source/geometries/Next100Shielding.cc

```diff
@@ -86,127 +85,122 @@ namespace nexus {
     this->SetLogicalVolume(lead_box_logic);
 
     //STEEL BEAM STRUCTURE
-    //auxiliar positions
-    G4double lat_beam_x = shield_x_/2.+ steel_thickness_ + lead_thickness_/2.;
-    G4double front_beam_z = shield_z_/2.+steel_thickness_+lead_thickness_/2.;
-    //   G4double frontz = front_beam_z - (roof_z_separation_+lateral_z_separation_/2.);
-    G4double top_beam_y = shield_y_/2.+ steel_thickness_ + lead_thickness_/2.;
-
-    G4Box* roof_beam = new G4Box("STRUCT_BEAM",lead_x_/2.,beam_base_thickness_/2.,lead_z_/2.);
-    G4Box* aux_box = new G4Box("AUX_box",shield_x_/2.+ steel_thickness_,beam_base_thickness_,shield_z_/2.+ steel_thickness_);
-    G4SubtractionSolid* roof_beam_solid = new G4SubtractionSolid("STRUCT_BEAM",roof_beam,aux_box,0,G4ThreeVector(0.,0.,0.));
-
-    G4Box* lat_beam_solid = new G4Box("STRUCT_BEAM",
-				      lead_thickness_/2.,
-				      shield_y_/2.+steel_thickness_/2.,
-				      beam_base_thickness_/2.);
+    // auxiliar positions used in translations
+    G4double lat_beam_x   = shield_x_/2. + steel_thickness_ + lead_thickness_/2.;
+    G4double front_beam_z = shield_z_/2. + steel_thickness_ + lead_thickness_/2.;
+    G4double top_beam_y   = (shield_y_-beam_thickness_)/2. + steel_thickness_ + beam_thickness_ + lead_thickness_/2.;
```

Comment:
Isn't it `top_beam_y   = shield_y_/2. + steel_thickness_ + beam_thickness_ + lead_thickness_/2.` ?

Reply:
No, because now the lead box center is "displaced" by `beam_thickness_/2` because the `lead_y_` dimension has and extra term of ` beam_thickness_`

Reply:
You are right, it's correct.